### PR TITLE
Adding new classes icon (for class selection dropdown)

### PR DESCRIPTION
**Pull request by capajon**
@njostonehouse 
Does this look like all that needs to be updated?  My build task flaked out on me so I ran it separately just for classes and pasted in the result... so definitely hoping for a sanity check.

Thanks for the help with this today too, btw. :)

### CODE REVIEW

File: tier3-icons.html [hunk 1]
```diff
@@ -31,6 +31,9 @@
...
+        </g>
```
Comment (njostonehouse) [inline tier3-icons.html]: I think the "fill" value is supposed to be cleared out of here.

File: images/tier3/classes.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
...488h-4.664zM13 3h4v4h-4V3zM12.668 11c-.462 0-.878.187-1.18.488-.3.302-.488.718-.488 1.18v4.664c0 .462.187.878.488 1.18.302.3.718.488 1.18.488h4.664c.462 0 .878-.187 1.18-.488.3-.302.488-.718.488-1.18v-4.664c0-.462-.187-.878-.488-1.18-.302-.3-.718-.488-1.18-.488h-4.664zM13 13h4v4h-4v-4zM22.668 21c-.462 0-.878.187-1.18.488-.3.302-.488.718-.488 1.18v4.664c0 .462.187.878.488 1.18.302.3.718.488 1.18.488h4.664c.462 0 .878-.187 1.18-.488.3-.302.488-.718.488-1.18v-4.664c0-.462-.187-.878-.488-1.18-.302-.3-.718-.488-1.18-.488h-4.664zM23 23h4v4h-4v-4zM22.668 1c-.462 0-.878.187-1.18.488-.3.302-.488.718-.488 1.18v4.664c0 .462.187.878.488 1.18.302.3.718.488 1.18.488h4.664c.462 0 .878-.187 1.18-.488.3-.302.488-.718.488-1.18V2.668c0-.462-.187-.878-.488-1.18-.302-.3-.718-.488-1.18-.488h-4.664zM23 3h4v4h-4V3zM22.668 11c-.462 0-.878.187-1.18.488-.3.302-.488.718-.488 1.18v4.664c0 .462.187.878.488 1.18.302.3.718.488 1.18.488h4.664c.462 0 .878-.187 1.18-.488.3-.302.488-.718.488-1.18v-4.664c0-.462-.187-.878-.488-1.18-.302-.3-.718-.488-1.18-.488h-4.664zM23 13h4v4h-4v-4z" color="#000" font-family="sans-serif" white-space="normal" overflow="visible" solid-color="#000000" fill="#72777a" fill-rule="evenodd"/>
```
Comment (njostonehouse) [inline images/tier3/classes.svg]: Some of the `style` values might not be necessary here.

Comment (capajon) [inline images/tier3/classes.svg]: Yeah you're definitely right, I should have noticed that.

**Comment by capajon** [comment]
Removed unnecessary styles.  Might be okay now

**Comment by njostonehouse** [comment]
That seems more like it.

**Pull request merged**